common/src/main/java/org/mcaccess/minecraftaccess/features/point_of_interest/LockingHandler.java

```diff
@@ -126,52 +128,69 @@ private void lookAtLockedTarget() {
      */
     private void bowAimingAssist() {
         LocalPlayer player = WorldUtils.getClientPlayer();
+        if (player == null) return;
+
+        // Check if player is using a bow
         if (config.aimAssistEnabled && !aimAssistActive && player.isUsingItem() && player.getUseItem().getItem() instanceof BowItem) {
             List<Entity> hostileEntities = BuiltinEntityPOIGroups.HOSTILE.group.getItems();
-            if (!hostileEntities.isEmpty()) {
-                Entity entity = hostileEntities.stream()
-                        .min(Comparator.comparingDouble(e -> WorldUtils.getClientPlayer().distanceTo(e)))
-                        .get();
-                if (lockOnEntity(entity)) {
-                    aimAssistActive = true;
+            if (hostileEntities != null && !hostileEntities.isEmpty()) {
```

Comment:
`hostileEntities` can never be null because the `getItems()` methods returns the result of a method annotated `@NotNull`.
```suggestion
            if (!hostileEntities.isEmpty()) {
```